sources2csr/ngs_maf_reader.py

```diff
@@ -0,0 +1,76 @@
+import csv
+import gzip
+import os
+from typing import Optional, Sequence, Dict, Any
+
+from sources2csr.ngs import NGS, LibraryStrategy
+from sources2csr.ngs_reader import NgsReader, NgsReaderException
+
+
+class GzipMafReaderException(Exception):
+    pass
+
+
+class GzipMafReader:
+    """ Reads gzipped MAF files.
+        MAF files, excluding the first row, are parsed as tab-separated text.
+    """
+    def __iter__(self):
+        return self.reader.__iter__()
+
+    def read_data(self) -> Sequence[Dict[str, Any]]:
+        data = []
+        header = None
+        for idx, line in enumerate(self):
+            if idx == 1:
```

Comment:
I think we should ignore all lines starting with `#` instead of relying on there always being a single comment line.
I don't know why people don't mention this kind of important details in describing file formats. I could only find the comment lines mentioned here: https://www.bx.psu.edu/~dcking/man/maf.xhtml, not here: https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#mutation-data or here: https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/

Reply:
Similarly for the txt files.

Reply:
good idea! I'll change this